src/common/create_component/script.py

```diff
@@ -170,11 +170,10 @@ def set_par_values(config) -> None:
       value = arg.get("default", arg.get("example", "..."))
     elif key == "de_train":
       value = "resources/neurips-2023-kaggle/de_train.parquet"
+    elif key == "de_train":
```

Comment:
Good point! However, do you mean de_train_h5ad?

Reply:
thanks for pointing out, just fixed